src/Core_Command.php

```diff
@@ -681,6 +685,22 @@ private function _multisite_convert( $assoc_args ) {
 			} else {
 				WP_CLI::warning( "Multisite constants could not be written to 'wp-config.php'. You may need to add them manually:" . PHP_EOL . $ms_config );
 			}
+		} else {
+			/**
```

Comment:
This is not a valid docblock, as it is not attached to an entity that can receive docblocks. Please use a regular multiline comment instead.